### PR TITLE
[RSDK-7818] impl camera_render_frame

### PR DESCRIPTION
**Pull request by mattjperez**
This PR adds the method needed to display an image on RC page in app.viam

### CODE REVIEW

File: micro-rdk/src/common/webrtc/grpc.rs [hunk 1]
```diff
@@ -21,8 +21,8 @@ use crate::{
 use super::{api::WebRtcError, sctp::Channel};
 
 #[cfg(feature = "camera")]
-// sizeof(fake_image) + 30bytes for headers
-static WEBRTC_GRPC_BUFFER_SIZE: usize = 10492;
+// sizeof(fake_image) + headers/encodings
+static WEBRTC_GRPC_BUFFER_SIZE: usize = 1024 * 11;
```
Comment (mattjperez) [inline micro-rdk/src/common/webrtc/grpc.rs:25]: running continuously on RC page revealed an occasional size of 10493 for `render_frame`. 
Bumped to the next largest multiple of 1024.

File: micro-rdk/src/common/camera/fake_camera.rs [hunk 2]
```diff
@@ -57,6 +57,16 @@ impl Camera for FakeCamera {
             .map_err(|_| CameraError::CameraGenericError("failed to encode GetImageResponse"))?;
         Ok(buffer)
     }
+    fn render_frame(&mut self, mut buffer: BytesMut) -> Result<BytesMut, CameraError> {
+        let msg = google::api::HttpBody {
+            content_type: "image/jpeg".to_string(),
+            data: FAKE_JPEG.into(),
+            extensions: Vec::new(),
```
Comment (npmenard) [inline micro-rdk/src/common/camera/fake_camera.rs:64]: nit: ..Default::default() is it's possible here

File: micro-rdk/src/common/camera/fake_camera.rs [hunk 3]
```diff
@@ -57,6 +57,16 @@ impl Camera for FakeCamera {
             .map_err(|_| CameraError::CameraGenericError("failed to encode GetImageResponse"))?;
         Ok(buffer)
     }
+    fn render_frame(&mut self, mut buffer: BytesMut) -> Result<BytesMut, CameraError> {
+        let msg = google::api::HttpBody {
+            content_type: "image/jpeg".to_string(),
+            data: FAKE_JPEG.into(),
```
Comment (npmenard) [inline micro-rdk/src/common/camera/fake_camera.rs:63]: is data a ref to an array or a vec?

File: micro-rdk/src/common/webrtc/grpc.rs (hunk 1 above)
Comment (npmenard) [reply micro-rdk/src/common/webrtc/grpc.rs:25]: do we know why?

**Review by npmenard — APPROVED** [review]
nit comments LGTM

File: micro-rdk/src/common/camera/fake_camera.rs (hunk 3 above)
Comment (mattjperez) [reply micro-rdk/src/common/camera/fake_camera.rs:63]: it's an owned vector, `pub data: ::prost::alloc::vec::Vec<u8>`. When removing `.into()`, compiler suggests `data: FAKE_JPEG.to_vec()`

File: micro-rdk/src/common/webrtc/grpc.rs (hunk 1 above)
Comment (mattjperez) [reply micro-rdk/src/common/webrtc/grpc.rs:25]: narrowing it down. something to do with the final encoded length.
I've verified that the sizeof `HttpBody` message is constant at `10477`, but the final encoded message length goes from `10492` to `10493` and continues at the latter from that point on (ran for 5min).

Comment (mattjperez) [reply micro-rdk/src/common/webrtc/grpc.rs:25]: should I make a ticket to investigate and move forward?

Comment (npmenard) [reply micro-rdk/src/common/webrtc/grpc.rs:25]: nope it's fine

File: micro-rdk/src/common/camera/fake_camera.rs (hunk 3 above)
Comment (npmenard) [reply micro-rdk/src/common/camera/fake_camera.rs]: ok make sense

**Pull request merged**